vars/zabbix.yml

```diff
@@ -16,6 +16,8 @@ sign_keys:
       sign_key: 79EA5ED4
     jessie:
       sign_key: 79EA5ED4
+    buster:
```

Comment:
Buster is already configured in this file.

Reply:
I can confirm that this does not work until the keys are added in like it is suggested.

Reply:
Ok, but if that is the case than the buster which is already configured has the wrong keys. So lets remove the incorrent ones and add the correct one.

Reply:
Correct signing key seems to A14FE591, fixed now.
Reason for double commit is because I patched the ansible galaxy published version - and then ported the changes to git master.

It would be good if a new release could be released and published to ansible galaxy so the debian 10 support gets published there.

Reply:
Thank you! Will make a release this weekend. It is long overdue...